fonts/Leipzig.svg

```diff
@@ -1,65 +1,12 @@
 <?xml version="1.0" standalone="no"?>
 <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd" >
-<!--
-2014-03-21: Created.
-Version 5.2.0 - adding glyphs (dynamics, ornaments, mensural note heads, etc)
-Version 5.2.1 - adding repeats
-Version 5.2.2 - fixing size of some glpyhs
-Version 5.2.3 - adding mensural clefs
-Version 5.2.4 - adding more dynam glyphs
-Version 5.2.5 - adding more dynam glyphs
-Version 5.2.6 - adding some octaves glyphs
-Version 5.2.7 - adjusting pedal glyphs
-Version 5.2.8 - adding perc clef
-Version 5.2.9 - adding two mensural clefs
-Version 5.2.10 - adding articulations and two fermatas and bow directions
-Version 5.2.11 - adding harmonic and reviewing bows
-Version 5.2.12 - reviewing artic width
-Version 5.2.13 - adding (very)short/long fermata and breath comma
-Version 5.2.14 - adding mensural stems
-Version 5.2.15 - changing ascent/descent to 750/250
-Version 5.2.16 - adding mensural rests
-Version 5.2.17 - correcting mensural rests position
-Version 5.2.18 - adding accid square brackets and reviewing bow directions
-Version 5.2.19 - adding some quartertone accidentals (Gould and Stein-Zimmermann)
-Version 5.2.20 - adding some anchor points (accidentals)
-Version 5.2.21 - adding stem flags (16th to 256th)
-Version 5.2.22 - fixing anchor point names
-Version 5.2.23 - adding anchor points to all accidentals
-Version 5.2.24 - changing 8th flags
-Version 5.2.25 - reviewing mensural clefs
-Version 5.2.26 - revising position of some rests
-Version 5.2.27 - adding some anchor points to note heads
-Version 5.2.28 - correcting anchor points
-Version 5.2.29 - adding plus time signature symbols
-Version 5.2.30 - adding some combining strokes for trills
-Version 5.2.31 - adding some arpegio glyphs (Mulit-segment lines)
-Version 5.2.32 - adding pedal sostenuto glyph
-Version 5.2.33 - adding mensural prolations
-Version 5.2.34 - adding some tablature glyphs
-Version 5.2.35 - adding 512th and 1024th flags
-Version 5.2.36 - adding 512th and 1024th rests
-Version 5.2.37 - adding some more tablature glyphs
-Version 5.2.38 - validating font
-Version 5.2.39 - adjusting some more tablature glyphs
-Version 5.2.40 - adding brace glyph
-Version 5.2.41 - adding sprechgesang glyph
-Version 5.2.42 - adding some notehead glyphs
-Version 5.2.43 - adding buzz roll glyph
-Version 5.2.44 - adding anchor points to noteheads
-Version 5.2.45 - adding precomposed ornaments
-Version 5.2.46 - adding single P and S for piano pedal
-Version 5.2.47 - adding glyphs for longer rests
-Version 5.2.48 - adding baroque ornament
-Version 5.2.49 - adding arabic accidentals
--->
```

Comment:
Please keep this

Reply:
It is, just not in the SVG. It's now in the FONTLOG in the sfd file, where it belongs and is written to FONTLOG.txt on regeneration.